server/spec/virt_spec.rb

```diff
@@ -252,7 +394,9 @@
 
   it 'should not bind products on host if virt_only are already available for guest' do
     @second_product = create_product(nil, nil, {:attributes => { :virt_only => true }})
-    create_pool_and_subscription(@owner['key'], @second_product.id, 10, [@virt_limit_product.id])
+    @cp.create_pool(@owner['key'],
```

Comment:
You need to make this the 2.0+ version in all places 
create_pool_and_subscription.